app/models/publications/monthly_statistics/by_status.rb

```diff
@@ -55,7 +55,7 @@ def formatted_counts
             case status
             when 'awaiting_provider_decision', 'interviewing'
               counts['Awaiting provider decisions'][phase] += count
-            when 'conditions_not_met', 'offer_deferred'
+            when 'conditions_not_met', 'offer_deferred', 'pending_conditions'
```

Comment:
I'm not sure whether mapping `conditions_not_met` or `offer_deferred` to `Conditions pending` makes sense. Deferrals are normally counted in the next year so I don't think these are relevant here. Conditions not met is a failure status so doesn't sit well with `offer accepted`.

Reply:
very good point